### DISCUSSION TITLE
https://fnoobt.github.io/posts/vps-v2ray-custom-route/

### DISCUSSION THREAD

**Discussion by giscus** [fnoobt/giscus#2] [Announcements]
# https://fnoobt.github.io/posts/vps-v2ray-custom-route/

目前，主流的V2Ray手机、电脑客户端都移除了对PAC模式的支持，这是因为通过功能强大的“（高级）路由设置”功能，同样可以实现PAC模式自动代理功能。

https://fnoobt.github.io/posts/vps-v2ray-custom-route/

**Comment by nidbCN** [comment]
提个建议：可以把data目录的链接换成：https://github.com/v2fly/domain-list-community/blob/master/README.md
否则该目录文件过多可能会查看困难

**Reply by fnoobt** [reply]
这是一个非常好的建议，非常感觉您的分享
